model_prices_and_context_window.json

```diff
@@ -382,6 +382,13 @@
         "litellm_provider": "openrouter",
         "mode": "chat"
     },
+    "openrouter/mistralai/mistral-7b-instruct": {
+        "max_tokens": 4096,
+        "input_cost_per_token": 0.0,
+        "output_cost_per_token": 0.0,
+        "litellm_provider": "openrouter",
```

Comment:
can you add cost per token

Reply:
![image](https://github.com/BerriAI/litellm/assets/25852482/5c8c26b9-e08f-4152-8dca-f682184e629c)

Price is actually zero on their website (see screenshot above). Pretty cool.